tests/test_merge_cdxj.py

```diff
@@ -0,0 +1,147 @@
+    assert original == MERGED_LIST
+    assert not non_edited
+    assert edited == 1
+    m_lang.assert_called_once_with(lang_array)
+
+
+@patch('merge_cdxj.get_sidecar_fields')
+def test_merge_meta_fields_non_edited(m_fields):
+    m_fields.return_value = {'url': 'http://www.example.com',
+                             'mime': 'text/html',
+                             'mime-detected': 'text/html',
+                             'charset': 'ascii',
+                             'languages': 'eng',
+                             'soft-404-detected': 0.08195022044249829}
+    cdxj = io.StringIO('com,example) 20091111212121 {"url": "http://www.example.com", \
+                                                     "mime": "text/html"}\n'
+                       'com,abc) 20091111212131 {"url": "http://www.abc.com", '
+                       '"mime": "text/xml"}\n', newline='\n')
+    merged_list = ['com,example) 20091111212121 {"url": "http://www.example.com", '
+                   '"mime": "text/html", "mime-detected": "text/html", '
+                   '"charset": "ascii", "languages": "eng", '
+                   '"soft-404-detected": 0.08195022044249829}\n',
+                   'com,abc) 20091111212131 {"url": "http://www.abc.com", "mime": "text/xml"}\n']
+    original, edited, non_edited = merge_cdxj.merge_meta_fields(META_DICT, cdxj)
+    assert original == merged_list
+    assert non_edited == 1
+    assert edited == 1
+    m_fields.assert_called_once()
+
+
+def test_create_dict_from_meta():
+    actual_dict = merge_cdxj.create_dict_from_meta(META_FILE)
+    assert actual_dict == META_DICT
+
+
+def test_create_cdxj_path(tmpdir):
+    cdxj_path = merge_cdxj.create_cdxj_path('home/file.cdxj', str(tmpdir))
+    expected_path = os.path.join(tmpdir / 'file_merged.cdxj')
+    assert cdxj_path == expected_path
+
+
+class Test_Merge_Cdxj:
+    @patch('merge_cdxj.create_dict_from_meta')
+    @patch('merge_cdxj.merge_meta_fields')
+    @patch('builtins.open', spec=open)
+    def test_merge_cdxj2(self, mock_open, m_merge_meta, m_create_dict, caplog, tmpdir):
+        caplog.set_level(INFO)
+        merged_file = io.StringIO()
+        m_create_dict.return_value = META_DICT
+        m_merge_meta.return_value = MERGED_LIST, 1, 0
+        handle1 = MagicMock()
+        handle1.__enter__.return_value = merged_file
+        handle1.__exit__.return_value = False
+        handle2 = MagicMock()
+        handle2.__enter__.return_value = META_FILE
+        handle2.__exit__.return_value = False
+        handle3 = MagicMock()
+        handle3.__enter__.return_value = CDXJ
+        handle3.__exit__.return_value = False
+        mock_open.side_effect = (handle1, handle2, handle3)
+        merge_cdxj.merge_cdxjs('meta.cdxj', 'original.cdxj', tmpdir)
+        assert 'Logging CDXJ merge information for original.cdxj and meta.cdxj' in caplog.text
+        assert 'Total edited records: %s', 1 in caplog.text
```

Comment:
This seems odd to me, is there a reason not to just hardcode "1" directly in the string?